.github/workflows/ci.yml

```diff
@@ -0,0 +1,129 @@
+name: CI
+on:
+  push:
+    branches: [staging, trying]
+  pull_request:
+    branches: [master]
+    types: [opened, reopened, synchronize]
+jobs:
+  test:
+    name: Tests
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [windows-latest, macos-latest, ubuntu-latest]
+        rust: [1.40.0]
+        target:
+          - i686-pc-windows-msvc
+          - x86_64-pc-windows-msvc
+          # - i686-pc-windows-gnu
+          # - x86_64-pc-windows-gnu
```

Comment:
Looks like there are some issues with compiling for MinGW using normal rust which should be fixed in 1.43.0 release coming up.

Reply:
Issues being? I'd like to dig into it.

Reply:
https://github.com/rust-lang/rust/pull/67429/